src/idiom162ex.fz

```diff
@@ -0,0 +1,10 @@
+ex162 is
+  for
+    arg in envir.args
+    first bool := true, false
+  do
+    if !first && arg = "b"
+      say "bat"
```

Comment:
This is not really what the spec says: _execute `bat` if `b` is a program option and `fox` if `f` is a program option._  My interpretation is that _any_ argument could be `b`or `f` to executed `bat`or `fox`, respectively.